lib/dor/services/client/object.rb

```diff
@@ -8,6 +8,7 @@ class Client
       # API calls that are about a repository object
       class Object < VersionedService
         extend Deprecation
+        self.deprecation_horizon = 'dor-service-client version 2.0'
```

Comment:
isn't this dor-services-client ?

Reply:
It is. I think this is telling upstream codebases that when dsc 2 comes out, affected methods will go away.

Reply:
dor-serviceSSSSS-client

Reply:
😆

Yes, totally a typo there!